tests/test_crm_dash.py

```diff
@@ -0,0 +1,459 @@
+
+    addr_family = 'ipv4'
+
+
+class Ipv6InboundRouting(InboundRouting):
+
+    addr_family = 'ipv6'
+
+
+class AclGroup(Resource):
+
+    addr_family = None
+
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+
+        self.acl_group_table = DashTable(self.dvs, swsscommon.APP_DASH_ACL_GROUP_TABLE_NAME)
+
+        self.counters = {
+            f'{self.addr_family}_acl_group': {'th_name': 'ACL_GROUP'},
+        }
+
+    def configure(self):
+        for i in range(1, self.high_th + 1):
+            self.acl_group_table.add(f"group{i}", {"ip_version": self.addr_family, "guid": f"dash-group-{i}"})
+
+    def clear(self):
+        self.acl_group_table.remove_all()
+
+
+class Ipv4AclGroup(AclGroup):
+
+    addr_family = 'ipv4'
+
+
+class Ipv6AclGroup(AclGroup):
+
+    addr_family = 'ipv6'
+
+
+class AclRule(Resource):
+
+    addr_family = None
+
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+
+        self.acl_group_table = DashTable(self.dvs, swsscommon.APP_DASH_ACL_GROUP_TABLE_NAME)
+        self.acl_rule_table = DashTable(self.dvs, swsscommon.APP_DASH_ACL_RULE_TABLE_NAME)
+
+        self.counters = {
+            f'{self.addr_family}_acl_rule': {'th_name': 'ACL_RULE', 'skip_counter': True},
+        }
+
+    def configure(self):
+        self.acl_group_table.add(f"group1", {"ip_version": self.addr_family, "guid": f"dash-group-1"})
+
+        for i in range(1, self.high_th + 1):
+            self.acl_rule_table.add(f"group1:rule{i}", {
+                "priority": i,
+                "action": "allow",
+                "terminating": "true",
+                "src_addr": f"{i}.0.0.0/0",
+                "dst_addr": f"{i}.0.0.0/0",
+                "src_port": "0-65535",
+                "dst_port": "0-65535"
+                })
+
+    def clear(self):
+        self.acl_rule_table.remove_all()
+        # Wait for the counters update by CrmOrch before removing the ACL group.
+        # If the ACL group will be removed immediately after removing its rules the CrmOrch
+        # won't send a "threshold cleared" message to Syslog.
+        time.sleep(self.polling_interval + 1)
+        self.acl_group_table.remove_all()
+
+
+class Ipv4AclRule(AclRule):
+
+    addr_family = 'ipv4'
+
+
+class Ipv6AclRule(AclRule):
+
+    addr_family = 'ipv6'
+
+
+class TestCrmDash:
+
+    @pytest.mark.parametrize('dash_entry', [
+        Vnet,
+        Eni,
+        Ipv4VnetMapping,
+        Ipv6VnetMapping,
+        Ipv4OutboundRouting,
+        Ipv6OutboundRouting,
+        Ipv4InboundRouting,
+        Ipv6InboundRouting,
+        Ipv4AclGroup,
+        Ipv6AclGroup,
+        Ipv4AclRule,
+        Ipv6AclRule
+    ])
+    def test_crm_dash_entry(self, dash_entry, dvs, testlog, dpu_only):
+        entry = dash_entry(dvs)
+        entry.set_tresholds()
+        entry.configure()
+        entry.check_treshold_exceeded()
+        entry.clear()
+        entry.check_treshold_cleared()
```

Comment:
Can we ensure all the configurations introduced as part of this test are cleaned up at the end?